### PR TITLE
Added an export form to the statistics page

### PR DESCRIPTION
**Pull request by voidus**
It exports raw bookings

### CODE REVIEW

**Comment by voidus** [comment]
https://mayflower-osl.atlassian.net/browse/PTC-20

File: phprojekt/htdocs/phpr/StatisticsView.js [hunk 1]
```diff
@@ -51,6 +59,28 @@ define([
         _setMonthGraph: function() {
             this.monthContainer.set('content', new monthGraph());
             this.monthViewState = 'graph';
+        },
+
+        _openExport: function(evt) {
+            if (evt) {
+                evt.stopPropagation();
+            }
+            debugger;
```
Comment (exi) [inline phprojekt/htdocs/phpr/StatisticsView.js]: i heard you like debuggers

Comment (voidus) [inline phprojekt/htdocs/phpr/StatisticsView.js]: whoops, fixed

File: phprojekt/application/Core/Views/scripts/index.csv.phtml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?php
+$df = fopen("php://output", 'w');
```
Comment (exi) [inline phprojekt/application/Core/Views/scripts/index.csv.phtml]: use all the exclamation marks

File: phprojekt/htdocs/css/themes/timecard/bookingList.css [hunk 3]
```diff
@@ -328,14 +328,14 @@
...
-.bookingCreator.bookingEditor .submitButton {
+.bookingEditor.bookingCreator .submitButton {
     visibility: hidden;
     height: 0;
     width: 0;
     overflow: hidden;
 }
 
-.bookingCreator .submitButton {
+form .submitButton {
```
Comment (exi) [inline phprojekt/htdocs/css/themes/timecard/bookingList.css]: is it wise to change that for any form with a '.submitButton' class?
I think a namespace would be wise

File: phprojekt/htdocs/css/themes/timecard/bookingList.css [hunk 4]
```diff
@@ -350,14 +350,14 @@
     padding: 2px;
 }
 
-.bookingCreater .submitButton:hover {
+form .submitButton:hover {
```
Comment (exi) [inline phprojekt/htdocs/css/themes/timecard/bookingList.css]: same as above

File: phprojekt/application/Core/Views/scripts/index.csv.phtml (hunk 2 above)
Comment (voidus) [inline phprojekt/application/Core/Views/scripts/index.csv.phtml]: Explain

File: phprojekt/htdocs/css/themes/timecard/bookingList.css (hunk 3 above)
Comment (voidus) [inline phprojekt/htdocs/css/themes/timecard/bookingList.css]: Yes, I think we should start to generalize design decisions like button style.

File: phprojekt/htdocs/css/themes/timecard/bookingList.css (hunk 4 above)
Comment (voidus) [inline phprojekt/htdocs/css/themes/timecard/bookingList.css]: see there

File: phprojekt/application/Core/Views/scripts/index.csv.phtml (hunk 2 above)
Comment (voidus) [inline phprojekt/application/Core/Views/scripts/index.csv.phtml]: ah, you meant quotation marks. Fixed.

File: phprojekt/htdocs/css/themes/timecard/bookingList.css (hunk 3 above)
Comment (voidus) [inline phprojekt/htdocs/css/themes/timecard/bookingList.css]: used class button instead of submitButton as discussed

**Pull request merged**